plugins/woocommerce/legacy/js/frontend/checkout.js

```diff
@@ -653,8 +653,8 @@ jQuery( function( $ ) {
 			});
 
 			var data = {
-				security: wc_checkout_params.remove_coupon_nonce,
-				coupon:   coupon
+				security:		wc_checkout_params.remove_coupon_nonce,
+				coupon_code:	coupon
```

Comment:
Let's revert this change, taking things back to:

```suggestion
				security: wc_checkout_params.remove_coupon_nonce,
				coupon:   coupon
```

Reason being this ajax handler expects to receive the `coupon` field in the request. Further down (line 671) we should switch from:

```js
$( document.body ).trigger( 'removed_coupon_in_checkout', [ data.coupon_code ] );
```

To:

```js
$( document.body ).trigger( 'removed_coupon_in_checkout', [ data.coupon ] );
```